src/guide/component-basics.md

```diff
@@ -243,6 +243,15 @@ Thanks to the `v-on:enlarge-text="postFontSize += 0.1"` listener, the parent wil
 </p>
 <script async src="https://static.codepen.io/assets/embed/ei.js"></script>
 
+We can list emitted events in the component's `emits` option:
+
+```js
+app.component('blog-post', {
+  props: ['title'],
+  emits: ['enlarge-text']
+})
+```
```

Comment:
might be good to explain why this is useful, i.e. being explicit to maintainers

Reply:
Added a note on emitted events validation 👍🏻